src/app/main/main.component.css

```diff
@@ -22,12 +22,22 @@
 }
 
 .drawer {
-    max-width: 40vw;
+    max-width: 30vw;
```

Comment:
I would suggest:

```
width: 40vw;
max-width: 600px;
```

^Where the 600px could be whatever width seems "not too wide" on a large monitor — I'm currently reviewing from a small laptop, where it won't get that large.